### PR TITLE
Feature/committee-dates-noah

### PR DESCRIPTION
**Pull request by noahmanger**

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -133,8 +133,21 @@ def currency_filter(num, grouping=True):
 def date_filter_sm(date_str):
     if not date_str:
         return ''
-    return parse_date(date_str).strftime('%m%y')
+    return parse_date(date_str).strftime('%m/%y')
 
+@app.template_filter('date_md')
```
Comment (jmcarp) [inline __init__.py]: Both newly added filters need unit tests. Tests can be added to `openfecwebapp/tests/test_utils.py` after #167 is merged.

File: __init__.py [hunk 2]
```diff
@@ -133,8 +133,21 @@ def currency_filter(num, grouping=True):
...
+def date_filter_md(date_str):
+    if not date_str:
+        return ''
+    return parse_date(date_str).strftime('%b %Y')
+
+@app.template_filter('last_3_characters')
+def last_3_characters(int):
```
Comment (jmcarp) [inline __init__.py]: How about a more concise implementation, e.g.
```python
def last_n_characters(value, nchar=3):
    if type(value) == int:
        return int % (10 ** nchar)
    return ''
```
If `nchar` is parameterized (I defaulted it to `3` above), then we can ask for arbitrary numbers of digits if we ever want to.

File: static/js/vendor/responsive-tables.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-$(document).ready(function() {
+jQuery(document).ready(function() {
```
Comment (msecret) [inline static/js/vendor/responsive-tables.js]: Why was this changed?

Comment (noahmanger) [inline static/js/vendor/responsive-tables.js]: Oh, I was confused because jQuery wasn't working but I forgot to run `npm install`. Reverting.
